src/main/java/io/github/syst3ms/skriptparser/expressions/ExprStringCases.java

```diff
@@ -0,0 +1,99 @@
+package io.github.syst3ms.skriptparser.expressions;
+
+import io.github.syst3ms.skriptparser.Main;
+import io.github.syst3ms.skriptparser.lang.Expression;
+import io.github.syst3ms.skriptparser.lang.TriggerContext;
+import io.github.syst3ms.skriptparser.parsing.ParseContext;
+import io.github.syst3ms.skriptparser.util.StringUtils;
+import org.jetbrains.annotations.Nullable;
+
+/**
+ * Converts a given string into a certain case type.
+ * Click <a href="https://www.chaseadams.io/posts/most-common-programming-case-types/">here</a> to learn about all these cases.
+ *
+ * @name Cased String
+ * @type EXPRESSION
+ * @pattern %strings% in (upper|lower)[ ]case
+ * @pattern %strings% in camel[ ]case
+ * @pattern %strings% in pascal case
+ * @pattern %strings% in snake case
+ * @pattern %strings% in kebab case
+ * @pattern %strings% in reverse[d] case
+ * @pattern mirror[ed] %strings%
+ * @since ALPHA
+ * @author Mwexim
+ */
+public class ExprStringCases implements Expression<String> {
+
+	static {
+		Main.getMainRegistration().addExpression(
+				ExprStringCases.class,
+				String.class,
+				false,
+				"%strings% in upper[ ]case",
```

Comment:
Uppercase and lowercase varients should have a pattern like `upper/lowercase %strings%`.

All other patterns should have a "to" alternative as well.

Reply:
I didn't do this because just handling the patterns is way easier than handling both patterns and parse marks.